src/test/java/io/cryostat/net/web/http/api/v2/RulesPostHandlerTest.java

```diff
@@ -218,6 +218,28 @@ void unsupportedFirstMimeShouldThrow() {
             MatcherAssert.assertThat(ex.getStatusCode(), Matchers.equalTo(415));
         }
 
+        @ParameterizedTest
+        @CsvSource(
```

Comment:
I think you can use `@ValueSource(strings={...})` in this case. It should do the same thing, but it's a bit more obvious what the intent is. `CsvSource` is good when you need to parameterize multiple arguments to the test method. Here we just need the single contentType parameter.